rotaract-appointments.php

```diff
@@ -225,7 +239,7 @@ function appointment_owners_field_cb( $args ) {
 	$options = get_option( 'rotaract_appointment_options' );
 	$owners  = get_all_owner();
 	?>
-	<select id="<?php echo esc_attr( $args['label_for'] ); ?>"
+	<select id="<?php esc_attr_e( $args['label_for'] ); ?>"
```

Comment:
Please revert this type of change due to `echo esc_attr` != `esc_attr_e`.

Reply:
[`esc_attr_e`](https://developer.wordpress.org/reference/functions/esc_attr_e/) == [`esc_attr__`](https://developer.wordpress.org/reference/functions/esc_attr__/) + `echo`
[`esc_attr_e`](https://developer.wordpress.org/reference/functions/esc_attr_e/) != [`esc_attr`](https://developer.wordpress.org/reference/functions/esc_attr__/) + `echo`